yt_dlp/extractor/vrt.py

```diff
@@ -425,3 +427,74 @@ def _real_extract(self, url):
                 ['description', 'twitter:description', 'og:description'], webpage),
             '_old_archive_ids': [make_archive_id('Canvas', video_id)],
         }
+
+
+class Radio1BeIE(VRTBaseIE):
+    _VALID_URL = r'https?://radio1\.be/\w+(/[\w-]+/[\w-]+)?/(?P<display_id>[\w-]+)'
```

Comment:
How about, like this?
```suggestion
    _VALID_URL = r'https?://radio1\.be/(?:lees|luister/select)/(?P<display_id>[\w/-]+)'
```